### ISSUE TITLE
Aggregation queries ignores pagination

### ISSUE THREAD

**Issue by marktani** [prisma/prisma#2897]
**Describe the bug**

Aggregation across nodes ignores pagination.

**To Reproduce**

1. Deploy this datamodel:

```
type User {
  id: ID! @unique
  name: String!
}
```

2. Create a few user nodes:

```
mutation a {
  createUser(data: {
    name: "Test"
  }) {
    id
  }
}
```
3. Count paginated users

```
query c {
  usersConnection(first: 3) {
    aggregate {
      count
    }
  }
}
```

4. See that count is not `3` but as many users as you created:

```
{
  "data": {
    "usersConnection": {
      "aggregate": {
        "count": 4
      }
    }
  }
}
```

**Expected behavior**

Should return the number of returned nodes:

```
{
  "data": {
    "usersConnection": {
      "aggregate": {
        "count": 3
      }
    }
  }
}
```

**Versions (please complete the following information):**
 - Prisma Server: `1.13.4`

**Comment by braska** [comment]
What the reason of this change? I am asking because this change broke my code.

When I implement pagination, I use `count` value. Before this change `count` was equal to total number of items. This value could be used to calculate total number of pages (`count / itemsPerPage`). But now I forced to use another one query to retrieve total number of items.

**Comment by mavilein** [comment]
@braska : We learned that the old behavior was often unintuitive to people. There we changed the behavior to return the number of nodes that match the specified filter (first, where...). If you are concerned about additional roundtrips i suggest to use aliases to send Prisma 2 queries at once. One for the results itself. And one for the total number of nodes.

```
query {
  results: usersConnection(first: 3) {
    aggregate {
      count
    }
  }
  totalCount: usersConnection {
    aggregate {
      count
    }
  }
}
```
